src/portal/src/i18n/lang/fr-fr-lang.json

```diff
@@ -287,8 +293,19 @@
         "NEW_MEMBER": "Nouveau membre",
         "MEMBER": "Membre",
         "NAME": "Nom",
-        "EMAIL": "Email",
         "ROLE": "Rôle",
+        "SYS_ADMIN": "Administrateur système",
+        "PROJECT_ADMIN": "Administrateur de projet",
+        "PROJECT_MAINTAINER": "Mainteneur du projet",
+        "DEVELOPER": "Développeur",
+        "GUEST": "Invité",
+        "LIMITED_GUEST": "Invité avec droits limités",
+        "DELETE": "Supprimer",
+        "ITEMS": "entrées",
```

Comment:
Sometimes item is translated by "élément" and sometimes by "entrée", we should be consistent I think

Reply:
Yes, élément seems more appropriate. I will correct it.

Reply:
I will change FILTER_ITEMS so all ITEMs are "entrée".